pkg/apis/sessions/session_state.go

```diff
@@ -69,6 +70,9 @@ func (s *SessionState) String() string {
 	if len(s.Groups) > 0 {
 		o += fmt.Sprintf(" groups:%v", s.Groups)
 	}
+	if s.IntrospectClaims != "" {
+		o += fmt.Sprintf(" Introspection Claims:%s", s.IntrospectClaims)
```

Comment:
Is this likely to contain sensitive information that we may not want logged?

Reply:
It will contain info like groups, roles and organizations, guess we can remove this log since the info is larger